projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-node.ts

```diff
@@ -7,13 +7,15 @@ import {
   SimpleChanges,
   TemplateRef,
   ViewChild,
-  ComponentRef
+  ComponentRef,
+  Directive
 } from '@angular/core';
 
 import { createValueForSchema, inferType, JSONEditorSchema } from './json-editor.helper';
 import { DialogComponent } from '../dialog/dialog.component';
 import { DialogService } from '../dialog/dialog.service';
 
+@Directive()
```

Comment:
I'm curious about why this is needed now.

Reply:
@duredhel13  Yes, I thought the same thing after updating. I believe this is why: https://angular.io/guide/migration-undecorated-classes

Reply:
Cool, thanks for the info.